newrelic-python.ini

```diff
@@ -32,6 +32,9 @@
 # https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key
 license_key = your_license_key
 
+# host
+host = data_posting_host
```

Comment:
we don't need this field since newrelic host is provided by ENV variable in automation

Reply:
Done